test/system_defs.jl

```diff
@@ -99,7 +99,7 @@ function barkley(T;
             Σ[i,j,r] = 0
         end
     end
-    function constant_step()
+    function constant_step!(u,v,Σ,a,b,Nx,Ny,s,r,D,Δt,h)
```

Comment:
You can probably save compile time by defining this _outside_ of the function (as a independent first level function).

Super minor though. Otherwise this is LGTM, the AppVeyor failure is unrelated. (nightlies...)